_config.yml

```diff
@@ -212,7 +212,7 @@ exclude:
   - bin
   - node_modules
   - vendor
-  - .git
+  - .github
```

Comment:
`.git` is a hidden directory and can contain sensitive info. shouldn't we continue excluding it?

Reply:
Actually, it doesn't matter because we already have S3 configured to redirect away from `.git/` for security purposes. Also, since none of the files in the `.git` directory include frontmatter, Jekyll won't include them in the output directory anyway. Disregard this comment :).